modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/upgrade/v8_9_1/test/CommerceChannelAccountEntryRelUpgradeProcessTest.java

```diff
@@ -0,0 +1,150 @@
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+import com.liferay.portal.upgrade.test.util.UpgradeTestUtil;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Balazs Breier
+ */
+@RunWith(Arquillian.class)
+public class CommerceChannelAccountEntryRelUpgradeProcessTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		Group group = GroupTestUtil.addGroup();
+
+		CommerceCurrency commerceCurrency =
+			CommerceCurrencyTestUtil.addCommerceCurrency(group.getCompanyId());
+
+		CommerceCatalog catalog =
+			_commerceCatalogLocalService.addCommerceCatalog(
+				null, RandomTestUtil.randomString(), commerceCurrency.getCode(),
+				LocaleUtil.US.getDisplayLanguage(), serviceContext);
+
+		CommerceChannel commerceChannel =
+			_commerceChannelLocalService.addCommerceChannel(
+				RandomTestUtil.randomString(),
+				AccountConstants.ACCOUNT_ENTRY_ID_DEFAULT, group.getGroupId(),
+				RandomTestUtil.randomString(),
+				CommerceChannelConstants.CHANNEL_TYPE_SITE, null,
+				commerceCurrency.getCode(), serviceContext);
+
+		CommercePriceList commercePriceList =
+			CommercePriceListTestUtil.addCommercePriceList(
+				catalog.getGroupId(), 0.0);
+
+		long userId = TestPropsValues.getUserId();
+
+		AccountEntry accountEntry =
+			CommerceAccountTestUtil.addBusinessAccountEntry(
+				userId, "Test Business Account", null, null,
+				new long[] {userId}, null, serviceContext);
+
+		_commerceChannelAccountEntryRel =
+			_commerceChannelAccountEntryRelLocalService.
+				addCommerceChannelAccountEntryRel(
+					userId, accountEntry.getAccountEntryId(),
+					CommerceDiscount.class.getName(),
+					commercePriceList.getCommercePriceListId(),
+					commerceChannel.getCommerceChannelId(), false,
+					RandomTestUtil.randomDouble(), RandomTestUtil.nextInt());
+	}
+
+	@Test
+	public void testUpdateCommerceChannelAccountEntryRelClassNameId()
+		throws Exception {
+
+		_runUpgrade();
+
+		EntityCacheUtil.clearCache();
+
+		CommerceChannelAccountEntryRel commerceChannelAccountEntryRel =
+			_commerceChannelAccountEntryRelLocalService.
+				getCommerceChannelAccountEntryRel(
+					_commerceChannelAccountEntryRel.
+						getCommerceChannelAccountEntryRelId());
+
+		Assert.assertEquals(
+			ClassNameLocalServiceUtil.getClassNameId(CommercePriceList.class),
```

Comment:
inject the Service, don't use the ServiceUtil (please check also in the other test if you did something like this)

Reply:
@smottal I was looking for this to fix it but forgot about it, in the newest PR I already used the injected service instead of the util.